### ISSUE TITLE
NullpointerException

### ISSUE THREAD

**Issue by masterofn1** [alamkanak/Android-Week-View#377]
Upon carefully following the given instructions, I added the gradle dependency. On the moment I am calling the view component on a layout file, I got a null pointer exception.

**Comment by entropitor** [comment]
Can you add steps to reproduce this? And preferably also some code

**Comment by masterofn1** [comment]
1. Import dependency via gradle
   compile 'com.github.alamkanak:android-week-view:1.2.6'
2. Add weekview on layout file

<com.alamkanak.weekview.WeekView
        android:id="@+id/weekView"
        android:layout_width="match_parent"
        android:layout_height="match_parent"
         ..........................................................

Then, I got these error
![image](https://cloud.githubusercontent.com/assets/10532443/19342944/a9c6ee84-9166-11e6-82fc-3ee59659a3c4.png)

**Comment by caske33** [comment]
Just to double check: does this also happen using the [Quivr-fork](https://github.com/Quivr/Android-Week-View)?

**Comment by CleytonGoncalves** [comment]
Exact same error here. And yes, also happens with the Quivr fork.

**Comment by EvanCooper9** [comment]
getting the same thing here unfortunately

**Comment by mzee96** [comment]
Also getting the same error here

**Comment by furqankhwaja** [comment]
Same issue. Pls give solutions soon

**Comment by camsiabor** [comment]
Is the problem related to 
WeekView.java line 969 -> throw new IllegalStateException("You must provide a MonthChangeListener")?

**Comment by Pralo** [comment]
I was able to solved this: 

You must provide a MonthChangeListener in which you should return at least an empty List of WeekViewEvent Class:

![screen shot 2016-10-26 at 10 28 14 am](https://cloud.githubusercontent.com/assets/4043802/19732652/04e9c93a-9b67-11e6-9833-f440c013f578.png)

**Comment by EvanCooper9** [comment]
@Pralo Yes you're right. In the example app, the `BaseActivity` class is meant to implement the MonthChangeListener method but never does (you can see that the method is actually implemented in the `BasicActivity` class.) **Having the MonthChangeListener in the right class fixes the issue.**

NOTE: You do get a Null Pointer Exception when looking at the XML in the editor, but it fixes it's self when you run on an emulator or a real device.

**Comment by lakshaya22** [comment]
Hello, can you specify the solution in some more detail, as I am having this issue still.

**Comment by dlucidone** [comment]
In the sample app change BasicActivity. java file content to the below content

   `import com.alamkanak.weekview.WeekViewEvent;`
   `import java.util.ArrayList;`
  ` import java.util.Calendar;`
   `import java.util.List;`
   `import com.alamkanak.weekview.MonthLoader;`

   `public class BasicActivity extends BaseActivity implements MonthLoader.MonthChangeListener {`
   ` @Override`
    `public List<? extends WeekViewEvent> onMonthChange(int newYear, int newMonth) {`
        `List<WeekViewEvent> events = new ArrayList<WeekViewEvent>();`
        `return events;`
   ` }`
`}`

Hope this Help!
